docs/getting_started.md

```diff
@@ -0,0 +1,166 @@
+
+The following command can be used to remove those cells: 
+
+```r
+# filter the `SingleCellExperiment`
+# the `$` notation denotes access to the colData slot of the `SingleCellExperiment` object 
+filtered_sce <- sce[, which(sce$miQC_pass)]
+```
+
+For more information on using `miQC` for filtering cells, see the following resources: 
+- [`miQC` vignette](https://bioconductor.org/packages/release/bioc/vignettes/miQC/inst/doc/miQC.html)
+- [Hippen _et al._ 2021](https://doi.org/10.1371/journal.pcbi.1009290)
+
+You can also directly filter cells based on the number of unique genes, total reads, and fraction of mitochondrial reads. 
+We used the function [`scuttle::addPerCellQCMetrics()`](https://rdrr.io/github/LTLA/scuttle/man/addPerCellQCMetrics.html) to calculate these metrics and have included them in the `colData` of the filtered `SingleCellExperiment` objects. 
+
+The following columns are added by `scuttle::addPerCellQCMetrics()` and can be found in the `colData`: 
+
+| Column name             | Contents                                                                                                                                                                                      |
+| ----------------------- | --------------------------------------------------------------------------------------------------------------------------|
+| `sum`                   | UMI count for RNA-seq data                                                                                                                           |
+| `detected`              | Number of genes detected (gene count > 0 )                                                                                                                                                    |
+| `subsets_mito_sum`      | UMI count of mitochondrial genes                                                                                                                                                              |
+| `subsets_mito_detected` | Number of mitochondrial genes detected                                                                                                                                                        |
+| `subsets_mito_percent`  | Percent of all UMI counts assigned to mitochondrial genes                                                                                                                                     |
+| `total`                 | Total UMI count for RNA-seq data and any alternative experiments (i.e., CITE-seq)                                                                                                             |
+
+These metrics can be used to directly filter the `SingleCellExperiment` object based on informed minimum thresholds. 
+
+If you are planning to filter low quality cells using these hard cutoffs, we encourage you to read more about the various metrics and plot the distribution of each metric before deciding on which cells to exclude. 
```

Comment:
Seemed like this could all be one paragraph. I also removed `minimum` since for at least one people are likely to use a max. 

```suggestion
These metrics can be used to directly filter the `SingleCellExperiment` object based on informed thresholds. 
If you are planning to filter low quality cells using such thresholds, we encourage you to read more about the various metrics and plot the distribution of each metric before deciding on which cells to exclude. 
```